app/views/setup/worker_bootstrap.html.slim

```diff
@@ -1,15 +1,34 @@
-h2 This is a Subtitle Just in Case We Need One
-p Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
-p The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.
+/
+  .alert.alert-info.alert-dismissible role="alert"
+    button.close[type="button" data-dismiss="alert" aria-label="Close"]
+      span[aria-hidden="true"]
+        | &times;
+    i.fa.fa-eye.fa-4x.pull-left aria-hidden="true"
+    span Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+
+h1 Bootstrap your Container as a Service Platform
+
+p
+  | In order to complete the installation, it is necessary to bootstrap few additional nodes, those will be the Kubernetes Master and Workers.
+  | This process is (almost) fully automated, and leverages on AutoYaST.
+  |  In case you are not familiar with it, you can find more information about AutoYaST in the
+
+  a href="https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html" official documentation.
+  |  The automatic installation gets invoked by adding autoyast=&lt;URL_TO_PROFILE&gt; to the kernel parameter list.
+  |  As installation media, you can use the very same image you bootstrapped the admin node with.
+
+  |  A ready to use AutoYaST profile has been already generated for you during the bootstrap of the admin node.
+  |  Bootstrap all the nodes you want to make part of this platform by adding the following boot parameter
+  |  autoyast=http://myserver/myconfig.xml
```

Comment:
I think it would be nice to have the actual link (the real name of this instance of the server) to show up there. Also it would be nice to make it a html link pointing to the autoyast file that was generated